hack/update-codegen.sh

```diff
@@ -4,13 +4,13 @@ set -o nounset
 set -o pipefail
 
 #deep copy
-$GOPATH/bin/deepcopy-gen --input-dirs "github.com/amadeusitgroup/workflow-controller/pkg/api/workflow/v1" \
+$GOPATH/bin/deepcopy-gen --input-dirs "github.com/amadeusitgroup/workflow-controller/pkg/api/workflow/v1,github.com/amadeusitgroup/workflow-controller/pkg/api/cronworkflow/v1" \
                          -p github.com/amadeusitgroup/workflow-controller/pkg/api/\
                          --bounding-dirs github.com/amadeusitgroup/workflow-controller/pkg/api \
                          -h "hack/boilerplate.go.txt"
```

Comment:
I understood why the generated files didn't contain the license: `boilerplate.go.txt` didn't contain it :)

Reply:
:1st_place_medal: mystery solved then